README.md

```diff
@@ -109,3 +109,15 @@ These are the arguments that apply to all commands:
     ```bash
     ./bin/odf -h
     ```
+
+### Download nightly build
+
+If you want to test the latest nightly build you can use the
+download-odf-nightly.sh script:
+
+```bash
+./download-odf-nightly.sh 29
+```
```

Comment:
```suggestion
```bash
./download-odf-nightly.sh 29
```
```

just wondering can we make this part of  new make command?